designs/virtualization/4.x/Templates-backed-by-golden-images/Templates backed by golden images.md

```diff
@@ -0,0 +1,30 @@
+---
+parent: Virtualization
+version: 4.x
+---
+# Templates backed by golden images
+
+Showing users which templates are backed by golden images (that are provided by OpenShift Virtualization/ or created by an admin).
+
+## Adding an ‘Operating system source’ column
+
+Add an ‘Operating system source’ column to the list view to show which Operating system is running on the VM and adding an icon to differentiate whether it's a vendor OS image or a custom OS image.
```

Comment:
Do we have a definition of what to show in the ‘Operating system source’ column ?
AFAIU, In the case of common templates, we don't have any data relating to the content of the base image (if it contains an OS or not, and what OS it contains in case it does contain an OS)
and In the case of regular templates (created by admin) we have no what to know if it has any base image or not.
what am I missing ?